pkg/operator/periodicbackupcontroller/periodicbackupcontroller_test.go

```diff
@@ -121,6 +123,77 @@ func TestSyncLoopWithDefaultBackupCR(t *testing.T) {
 	require.Equal(t, expDisabledBackupVar, controller.backupVarGetter.ArgString())
 }
 
+func TestSyncLoopFailsDegradesOperatorWithDefaultBackupCR(t *testing.T) {
```

Comment:
This e2e scenario 

- assert the `etcd-backup-server` container being disabled by default
- assert the `etcd-backup-server` container being enabled upon installing `Backup` CR with `name=default`
- the fake etcd pods being used has the `etcd-backup-server` container failing
- assert the operator status is degraded 

afterwards 

- delete `Backup` CR with `name=default`
-  assert the `etcd-backup-server` container being disabled 
- - assert the operator status is back to normal

cc @openshift/openshift-team-etcd  @vrutkovs